src/traces/histogram/attributes.js

```diff
@@ -124,14 +119,7 @@ module.exports = {
     },
     ybins: makeBinsAttr('y'),
 
-    marker: extendFlat({}, {
-        showscale: barMarkerAttrs.showscale,
-        line: extendFlat({},
-            {width: extendFlat({}, barMarkerLineAttrs.width, {dflt: 0})},
```

Comment:
The histogram attribute had `dflt: 0` even before this PR, explaining why this bug didn't extend to histogram traces.